tianshou/highlevel/experiment.py

```diff
@@ -114,6 +114,22 @@ class ExperimentConfig:
     policy_persistence_mode: PolicyPersistence.Mode = PolicyPersistence.Mode.POLICY
     """Controls the way in which the policy is persisted"""
 
+@dataclass
+class EvaluationProtocalExperimentConfig(ExperimentConfig):
+    train_seed_mechanism: Literal["consecutive"]|Literal["repeat"] = "consecutive" # or repeat, if all train seeds are supposed to be the same or consecutive numbers, compare seed function in venvs.py
```

Comment:
Literal["consecutive", "repeat"] would be better, but let's instead introduce an enum

Instead of block comment, write a docstring, like in the other classes